<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myMainWindow</class>
 <widget class="QMainWindow" name="myMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>myMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QmyVideoWidget" name="videoWidegt" native="true">
      <property name="minimumSize">
       <size>
        <width>563</width>
        <height>385</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnAdd">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="image.qrc">
          <normaloff>:/img/image/open.jfif</normaloff>:/img/image/open.jfif</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPlay">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="image.qrc">
          <normaloff>:/img/image/bofang.jfif</normaloff>:/img/image/bofang.jfif</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPause">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="image.qrc">
          <normaloff>:/img/image/zanting.jfif</normaloff>:/img/image/zanting.jfif</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStop">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="image.qrc">
          <normaloff>:/img/image/tingzhi.jpg!ys</normaloff>:/img/image/tingzhi.jpg!ys</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>96</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnSound">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnFullScreen">
        <property name="text">
         <string>全屏</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="labCurMedia">
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderPosition">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labRatio">
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmyVideoWidget</class>
   <extends>QWidget</extends>
   <header>QmyVideoWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
